package.json

```diff
@@ -5,17 +5,21 @@
   "proxy": "https://acmucsd-membership-portal-api.herokuapp.com",
   "dependencies": {
     "@ant-design/icons": "^4.0.6",
+    "@firebase/app": "^0.6.3",
```

Comment:
Why is there a firebase bump here

Reply:
It marked it as an implied dependency and required that it was present in the package.json.